session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java

```diff
@@ -59,7 +60,7 @@
 class NotificationExpirationManagement implements RedisExpirationStrategy {
   static Logger logger = LoggerFactory.getLogger(NotificationExpirationManagement.class);
 
-  private static final long ONE_MINUTE = TimeUnit.MINUTES.toMillis(1);
+  private static final long ONE_MINUTE_IN_SECOND = TimeUnit.MINUTES.toSeconds(1);
```

Comment:
Keep ONE_MINUTE as constant name.

Reply:
ok, if you want , but it takes me timje to unserstand that it is a minute in second units :-)